lib/structures/User.js

```diff
@@ -46,6 +63,13 @@ class User extends Base {
              */
             this.avatar = data.avatar;
         }
+        if(data.avatar_decoration_data !== undefined) {
+            /**
+             * The data of the user's avatar decoration, or null if no avatar decoration
+             * @type {Object}
```

Comment:
It would be probably worth to declare the field as nullable, possibly even its own typedef at the end of the file?
```suggestion
             * @type {Object?}
```